### PR TITLE
Trdt 648 649

### PR DESCRIPTION
**Pull request by basotia**
CIP insights integration updated unit test

### CODE REVIEW

File: app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala [hunk 1]
```diff
@@ -43,36 +43,15 @@ class InsightConnector @Inject()(val httpClient: HttpClient,
           case OK =>
             val fraudRiskReport = response.json.validate[FraudRiskReport].get
```
Comment (adamconder) [inline app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala:44]: Should we be doing .get here? Can we fold/pattern match and handle the result?

File: app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala [hunk 2]
```diff
@@ -43,36 +43,15 @@ class InsightConnector @Inject()(val httpClient: HttpClient,
...
             Right(ResponseWrapper(correlationId,fraudRiskReport))
-          case BAD_REQUEST =>
+          case _ =>
+            logger.error(s"$correlationId::[InsightConnector:assess]Unable to get fraudrisk report as unknown code returned ${response.status} returned, with message ${response.body}")
```
Comment (adamconder) [inline app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala:47]: Do we have guarantees what's going to be in this response body? Could it contain PII or sensitive data?

File: test/uk/gov/hmrc/transactionalrisking/v1/mocks/connectors/MockInsightConnector.scala [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+trait MockInsightConnector extends MockFactory with MockAppConfig {
+
+  val mockInsightConnector: InsightConnector = mock[InsightConnector]
+  var port: Int = _
+  val cipFraudServiceBaseUrl: String = s"http://localhost:$port/fraud"
+
+  object MockInsightConnector {
+
+    //MockedAppConfig.cipFraudServiceBaseUrl returns cipFraudServiceBaseUrl
```
Comment (adamconder) [inline test/uk/gov/hmrc/transactionalrisking/v1/mocks/connectors/MockInsightConnector.scala:37]: Can we remove commented code

File: app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala (hunk 2 above)
Comment (basotia) [reply app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala]: removed

File: app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala (hunk 1 above)
Comment (basotia) [reply app/uk/gov/hmrc/transactionalrisking/v1/services/cip/InsightConnector.scala]: done

File: test/uk/gov/hmrc/transactionalrisking/v1/mocks/connectors/MockInsightConnector.scala (hunk 3 above)
Comment (basotia) [reply test/uk/gov/hmrc/transactionalrisking/v1/mocks/connectors/MockInsightConnector.scala]: removed

**Review by adamconder — APPROVED** [review]

**Pull request merged**